test/FunctionalTests/Microsoft.OData.Client.Tests/Serialization/MockHttpClientHandler.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.OData.Client.Tests.Serialization
     /// A mock implementation of <see cref="HttpClientHandler"/>
     /// for testing purposes.
     /// </summary>
-    internal sealed class MockHttpClientHandler : HttpClientHandler
+    internal class MockHttpClientHandler : HttpClientHandler

```

Comment:
Is this class used anywhere since we did away with the `HttpClientHandler`?

Reply:
Yes, it's used in tests, specifically it's used to created mock `HttpClientHandler` implementations that are passed to the `MockHttpClientFactory` to test things like timeouts, request cancellation, response handling, etc.